frontend/javascripts/admin/time/time_line_chart_view.js

```diff
@@ -0,0 +1,91 @@
+// @flow
+import { Chart } from "react-google-charts";
+import * as React from "react";
+
+export type ColumnDefinition = {
+  id?: string,
+  type: string,
+  role?: string,
+  p?: Object,
+};
+
+export type RowContent = [string, string, string, Date, Date];
+
+export type DateRange = [moment$Moment, moment$Moment];
+
+type Props = {
+  columns: Array<ColumnDefinition>,
+  rows: Array<RowContent>,
+  timeAxisFormat: string,
+  dateRange: DateRange,
+};
+
+export default class TimeTrackingChart extends React.PureComponent<Props> {
+  additionalCSS: ?HTMLStyleElement = null;
+  chartScrollElement: ?HTMLElement = null;
+
+  componentWillUnmount() {
+    if (this.chartScrollElement) {
+      this.chartScrollElement.removeEventListener("mousemove", this.adjustTooltipPosition);
+    }
+  }
+
+  // We need to adjust the tooltips position manually because it is not positioned correctly when scrolling down.
+  // This fix was suggested by
+  // https://stackoverflow.com/questions/52755733/google-charts-tooltips-have-wrong-position-when-inside-a-scrolling-container.
+  applyTooltipPositioningFix = () => {
```

Comment:
Did you adapt the fix now? Last time we talked, there was still a problem with the fix, right? Would be cool if you could add a code comment which describes briefly what you changed to make it work :)

Reply:
I added some more comments on that. I hope they are precise enough and not too long